standalone/src/test/java/io/pravega/local/TlsEnabledInProcPravegaClusterTest.java

```diff
@@ -115,9 +117,17 @@ public void testWriteAndReadEventWhenConfigurationIsProper() throws ExecutionExc
 
         // Keeping the read timeout large so that there is ample time for reading the event even in
         // case of abnormal delays in test environments.
-        EventRead<String> event = reader.readNextEvent(150000);
+        long readTimeOut = 10000;
+
+        EventRead<String> event = reader.readNextEvent(readTimeOut);
+        if (event.isCheckpoint()) {
```

Comment:
Shouldn't you do this in a loop? It is unlikely, but I suppose you can get multiple checkpoint events.

Reply:
@fpj I considered it but then thought that the likelihood of occurrence of that scenario is too low (negligible, I think) to make the test any more complex. All that the test is intended to do is to write an event, read it back and compare what was read with what was written (and do all these while SSL/TLS is enabled). An extra loop IMHO will add a few inessential/boilerplate elements and make it harder to understand the true underlying purpose of the test.  That being said, I'll defer to your opinion on this. What do you say?

In fact, I was thinking of creating an issue for changing the client API to have a way to read without needed to tackle checkpoint events by default. It's unintuitive to have to deal with checkpoint events mandatorily when reading events. If we agree on that, there'd be no need for even checking if an event is a checkpoint event at all.

Reply:
Checkpoint events are necessary for correctness, but let's not talk about it here. As I was going through your response, it occurred to me that you can simply disable automatic checkpointing for the sake of this test case. Do it in the reader group config on line 105, pls.

Reply:
Done. The test has become much simpler now! Thanks.